spec/controllers/application_letters_controller_spec.rb

```diff
@@ -128,12 +132,6 @@
         expect(assigns(:application_letter)).to eq(application)
       end
 
-      it "redirects to the application" do
```

Comment:
Did you also write a test for that? That it redirects to the overview page? (Can't find any here)